lib/qa/authorities/linked_data/config/search_config.rb

```diff
@@ -105,7 +105,7 @@ def results_altlabel_predicate
       # Does this authority configuration support sorting of search results?
       # @return [True|False] true if sorting of search results is supported; otherwise, false
       def supports_sort?
-        return true unless results_sort_ldpath.present? || !results_sort_predicate.present?
+        return true if results_sort_ldpath.present? || results_sort_predicate.present?
```

Comment:
Shifting lod_full_config fixture to specify ldpath for sort instead of a predicate revealed the logic on this was incorrect.  The test now verifies that if sort ldpath is present, then sorting is supported.